Code/IO/src/sitkImageViewer.cxx

```diff
@@ -152,66 +167,66 @@ void ImageViewer::initializeDefaults()
     {
     if ( itksys::SystemTools::GetEnv ( win_dirs[i], ProgramFiles ) )
       {
-      m_GlobalDefaultSearchPath.push_back ( ProgramFiles + "\\" );
+      m_DefaultSearchPath.push_back ( ProgramFiles + "\\" );
       }
     }
 
   if ( itksys::SystemTools::GetEnv ( "USERPROFILE", ProgramFiles ) )
     {
-    m_GlobalDefaultSearchPath.push_back ( ProgramFiles + "\\" );
-    m_GlobalDefaultSearchPath.push_back ( ProgramFiles + "\\Desktop\\" );
+    m_DefaultSearchPath.push_back ( ProgramFiles + "\\" );
+    m_DefaultSearchPath.push_back ( ProgramFiles + "\\Desktop\\" );
     }
 
 #elif defined(__APPLE__)
 
   // Common places on the Mac to look
-  m_GlobalDefaultSearchPath = { "/Applications/", "/Developer/", "/opt/", "/usr/local/" };
+  m_DefaultSearchPath = { "/Applications/", "/Developer/", "/opt/", "/usr/local/" };
   std::string homedir;
   if ( itksys::SystemTools::GetEnv ( "HOME", homedir ) )
     {
-    m_GlobalDefaultSearchPath.push_back( homedir + "/Applications/" );
+    m_DefaultSearchPath.push_back( homedir + "/Applications/" );
     }
 
 
 #else
 
   // linux and other systems
-  m_GlobalDefaultSearchPath = { "./" };
+  m_DefaultSearchPath = { "./" };
   std::string homedir;
   if ( itksys::SystemTools::GetEnv ( "HOME", homedir ) )
     {
-    m_GlobalDefaultSearchPath.push_back( homedir + "/bin/" );
+    m_DefaultSearchPath.push_back( homedir + "/bin/" );
     }
 
-  m_GlobalDefaultSearchPath.emplace_back("/opt/" );
-  m_GlobalDefaultSearchPath.emplace_back("/usr/local/" );
+  m_DefaultSearchPath.emplace_back("/opt/" );
+  m_DefaultSearchPath.emplace_back("/usr/local/" );
 
 #endif
 
-  localDebugMacro( << "Default search path: " << m_GlobalDefaultSearchPath << std::endl );
+  //localDebugMacro( << "Default search path: " << m_DefaultSearchPath << std::endl );
```

Comment:
Maybe add logic to the getInstance which prints the default configuration the first time its called in debug mode.

Reply:
No, that can't happen.  Debug mode can't happen the first time getInstance is called.  At the start debug is false.  To set it to true, you'd have to call SetGlobalDefaultDebug, which calls getInstance already.  I'll make it so if you call SetGlobalDefaultDebug with true, it prints config.